src/GenAPI/genapi.slnf

```diff
@@ -1,17 +0,0 @@
-{
-  "solution": {
-    "path": "..\\..\\sdk.sln",
-    "projects": [
-      "src\\GenAPI\\Microsoft.DotNet.GenAPI\\Microsoft.DotNet.GenAPI.csproj",
-      "src\\GenAPI\\Microsoft.DotNet.GenAPI.Task\\Microsoft.DotNet.GenAPI.Task.csproj",
-      "src\\GenAPI\\Microsoft.DotNet.GenAPI.Tool\\Microsoft.DotNet.GenAPI.Tool.csproj",
-      "src\\Cli\\Microsoft.DotNet.Cli.Utils\\Microsoft.DotNet.Cli.Utils.csproj",
-      "src\\Cli\\Microsoft.DotNet.InternalAbstractions\\Microsoft.DotNet.InternalAbstractions.csproj",
-      "src\\Layout\\toolset-tasks\\toolset-tasks.csproj",
-      "src\\Microsoft.DotNet.ApiSymbolExtensions\\Microsoft.DotNet.ApiSymbolExtensions.csproj",
-      "src\\Tests\\Microsoft.DotNet.GenAPI.Tests\\Microsoft.DotNet.GenAPI.Tests.csproj",
-      "src\\Tests\\Microsoft.DotNet.ApiSymbolExtensions.Tests\\Microsoft.DotNet.ApiSymbolExtensions.Tests.csproj",
-      "src\\Tests\\Microsoft.NET.TestFramework\\Microsoft.NET.TestFramework.csproj",
-    ]
-  }
-}
```

Comment:
That file was actually quite nice. Feel free to bring it back if you want to. We have the same for apicompat.

Reply:
Unfortunately, loading this file into VS kept crashing it, so I opted to remove it.
It doesn't crash for apicompat.slnf, so I'd have to figure out if something's missing.